examples/water/dpa3/run.sh

```
@@ -0,0 +1,23 @@
+# unset PADDLE_TRAINER_ENDPOINTS
+# unset DISTRIBUTED_TRAINER_ENDPOINTS
+# unset FLAGS_START_PORT
+# unset PADDLE_ELASTIC_TIMEOUT
+# export NNODES=1
+# export PADDLE_TRAINERS_NUM=1
+# unset CUDA_DEVICE_MAX_CONNECTIONS
+
+
+rm -rf output/
+
+${nsys_args} python -m paddle.distributed.launch \
```

Shellcheck (0.10.0)

[warning] nsys_args is referenced but not assigned.

(SC2154)